portfolio/src/main/webapp/script.js

```diff
@@ -52,6 +52,18 @@ function getRandomItem(curr, arr) {
 
 function getData() {
   fetch('/data').then(response => response.text()).then((data) => {
-    document.getElementById('data-container').innerHTML = data;
+    let dataJson = JSON.parse(data);
+    let comments = dataJson.comments;
+    let maxNum = dataJson.maxNumOfComments;
+    document.getElementById('data-container').innerText = comments;
```

Comment:
innerText doesn't feel right here. You should have a renderer for each comment with divs and stuff.

Reply:
Yep, I changed it to render the comments as a list